### PR TITLE
Add conditional youtube subtitle support

### PR DESCRIPTION
**Pull request by jgmize**
@glogiotatidis r?

### CODE REVIEW

File: masterfirefoxos/base/utils.py [hunk 1]
```diff
@@ -61,3 +64,15 @@ def copy_tree(page):
...
+
+
+def youtube_embed_url(request, en_youtube_id):
+    embed = 'https://www.youtube.com/embed/'
+    youtube_id = _(en_youtube_id)
+    if youtube_id == en_youtube_id and request and (
+            not request.path.startswith('/en/')):
+        query_template = '?hl={lang}&cc_lang_pref={lang}&cc_load_policy=1'
+        lang = request.path.split('/')[1]
+        if lang in settings.LANGUAGE_NAMES:
```
Comment (glogiotatidis) [inline masterfirefoxos/base/utils.py]: I believe that this is not needed. If we reach `render()` then `lang` will always be in `settings.LANGUAGE_NAMES`

**Comment by glogiotatidis** [comment]
Looks good! r+wc

File: masterfirefoxos/base/utils.py (hunk 1 above)
Comment (jgmize) [inline masterfirefoxos/base/utils.py]: I agree, that *should* be the case, I was just being extra paranoid here.

**Comment by jgmize** [comment]
Thanks @glogiotatidis, I removed the extraneous lang validation and associated test.

**Pull request merged**